scripts/main.js

```diff
@@ -146,9 +138,18 @@ $(function(){
         function addRow(hideDeleteButton) {
             var $row = $(rowTemplate.replace(/{{type}}/g, type).replace(/{{id}}/g, rowID++));
 
-            $row.find('.input--date').attr('min', yyyymmdd());
-
-            if (!hideDeleteButton && Modernizr.inputtypes.date) {
+            var userLanguage = navigator.language || navigator.userLanguage; 
+            $.datepicker.setDefaults($.datepicker.regional[userLanguage]);
+            $row.find('.input--date').datepicker({
+                altField: "#alt-"+$row.find('.input--date').attr('id'), 
+                altFormat: "yy-mm-dd", 
+                yearRange: '2016:2016',
+                minDate: 0, 
+                maxDate: new Date(2016,11 -1,08)})
+            .on('keypress', function(e){ e.preventDefault(); })
+            .on('paste', function(e){ e.preventDefault(); });
```

Comment:
As above with the conditional usage: Can we ensure that this is only applied when `!Modernizr.inputtypes.date`?